src/pages/SyntheticsPage/SyntheticsPage.tsx

```diff
@@ -426,7 +428,13 @@ export function SyntheticsPage(p: Props) {
                 ordersData={ordersInfoData}
                 isLoading={isPositionsLoading}
                 savedIsPnlInLeverage={savedIsPnlInLeverage}
-                onOrdersClick={() => setListSection(ListSection.Orders)}
+                onOrdersClick={(key) => {
+                  setListSection(ListSection.Orders);
+                  setSelectedPositionOrderKey(key);
+                  if (key) {
+                    setSelectedOrdersKeys((prev) => ({ ...prev, [key]: true }));
+                  }
+                }}
```

Comment:
Just a nitpick but maybe move to function like handleOrdersClick. Here and in the latter place.